APIs/IPMXAPI.raml

```diff
@@ -0,0 +1,91 @@
+#%RAML 1.0
+
+title: IPMX Device Control
+baseUri: http://example.api.com/x-nmos/ipmx/{version}
```

Comment:
Perhaps the scope of the API needs to be reduced to match the scope of the repository? I know it difficult to talk about this without including IPMX but the API is specific to EDID devices?

Reply:
Suggest we use Peter's placeholder IS-EDID-CM for now until there is an official IS-XX.  Scope we have for now from Steering is EDID type "control" so better to avoid confusions on the API. Is it possible than an IPMX Device Controller that provides an IS-EDID-CM API could in the future provide other APIs? How the other APIs for control come on line can influence this also I think,

Reply:
> Is it possible than an IPMX Device Controller that provides an IS-EDID-CM API could in the future provide other APIs?

Absolutely, for control API we currently have IS-05 and IS-08... maybe 7? 🤔 
I expect that list to grow as well.

> How the other APIs for control come on line can influence this

💯 We need to take all the use cases into account but we should not force every device/appliance to have one monolithic API